src/System.Net.HttpListener/src/System/Net/Windows/AsyncRequestContext.cs

```diff
@@ -14,6 +14,22 @@ internal sealed unsafe class AsyncRequestContext : RequestContextBase
         private ThreadPoolBoundHandle _boundHandle;
         private readonly ListenerAsyncResult _result;
 
+#if DEBUG
+        private volatile int _nativeOverlappedCounter = 0;
+        private volatile int _nativeOverlappedUsed = 0;
+
+        private void ReleaseNativeOverlapped()
```

Comment:
I'd prefer we call this, DEBUG-only, method something like "TrackReleaseNativeOverlapped" or "DebugTrackReleaseNativeOverlapped".  To me, it looks weird to have a name that seems like the code shouldn't be DEBUG-only.